src/register/RegistrationPage.jsx

```diff
@@ -145,7 +145,8 @@ const RegistrationPage = (props) => {
       setErrorCode({ type: '', count: 0 });
       const nextStep = getMultiStepRegistrationNextStep(multiStepRegistrationPageStep);
       if (nextStep === SECOND_STEP) {
-        trackMultiStepRegistrationStep2Viewed(multiStepRegistrationExpVariation);
+        const isMarketingLead = formFields.email && configurableFormFields?.marketingEmailsOptIn;
```

Comment:
i know `formFields.email &&` is useless here but i still added for logical sense.